### PR TITLE
Allow Configurator unittest to be run from any directory.

### PR DESCRIPTION
**Pull request by ewiseblatt**
@lwander

### CODE REVIEW

File: unittest/configurator_test.py [hunk 1]
```diff
@@ -23,18 +23,30 @@
...
+                               'spinnaker-local.yml'), 'w') as f:
+          f.write('empty:\n')
+
+        # We arent using HOME in a test, but want to verify that it is
+        # being used.
+        old_home = os.environ['HOME']
+        os.environ['HOME'] = temp_homedir
+        try:
+            configurator = Configurator()
+        finally:
```
Comment (lwander) [inline unittest/configurator_test.py]: Does this catch signals that could kill the test before restoring `$HOME`?

File: unittest/configurator_test.py [hunk 2]
```diff
@@ -99,5 +114,14 @@ def test_update_deck_settings(self):
...
+
+  # The configurator requires we be run in the build directory
+  # containing deck. So let's pretend this was the case.
+  temp_indir = tempfile.mkdtemp()
+  os.environ['PWD'] = temp_indir
+  os.mkdir(os.path.join(temp_indir, 'deck'))
+  try:
+      unittest.TextTestRunner(verbosity=2).run(suite)
+  finally:
+      shutil.rmtree(temp_indir)
```
Comment (lwander) [inline unittest/configurator_test.py]: Is it OK to remove `$PWD`?

File: unittest/configurator_test.py (hunk 1 above)
Comment (ewiseblatt) [inline unittest/configurator_test.py]: Why is that a concern here?

Comment (lwander) [inline unittest/configurator_test.py]: Canceling the test with `^C` could leave you with a broken home directory.

Comment (ewiseblatt) [inline unittest/configurator_test.py]: Subprocess environments shouldnt leak up to the parent's environment.

Comment (lwander) [inline unittest/configurator_test.py]: Ah, right

**Comment by anotherchrisberry** [comment]
@ewiseblatt @lwander is this PR still active or can it be closed?

**Comment by ewiseblatt** [comment]
submitted. thansk for the headds up.

**Pull request merged**